README.md

```diff
@@ -32,7 +32,7 @@ See the [open issues](#) for a full list of proposed features (and known issues)
 
 
 ## Set up
```

Comment:
@klml "Set up" is ambigious:

- setting up a workspace environment to be able to develop/contribute (make) or
- use/ run / deploy the project (helm)

i think it would be better to seperate those two.